lib/v2/tg/client.js

```diff
@@ -0,0 +1,194 @@
+const readline = require('node:readline/promises');
+const { TelegramClient } = require('telegram');
+const { StringSession } = require('telegram/sessions');
+const { Api } = require('telegram/tl/api');
+
+const config = require('../../config').value.tg;
+
+const apiId = config.apiId ?? 21834366;
+const apiHash = config.apiHash ?? 'd9601f8a2762a03e910a8cfc2ba79004';
```

Comment:
Do not provision your own api id here. You may include the [official one](https://github.com/DrKLO/Telegram/blob/master/TMessagesProj/src/main/java/org/telegram/messenger/BuildVars.java).